src/clj/game/core/runs.clj

```diff
@@ -350,12 +350,23 @@
   ;; clears the broken subs out of the prompt, otherwise they can get stuck with some cards
   (reset-all-subs! state (get-card state ice))
   (show-run-prompts state (str "encountering " (:title ice)) ice)
-  (wait-for (encounter-ice state side (make-eid state eid) ice)
-            (clear-run-prompts state)
-            (if (and (not (:run @state))
-                     (empty? (:encounters @state)))
-              (forced-encounter-cleanup state :runner eid)
-              (effect-completed state side eid))))
+  ;; do we need to mess with the run state
+  (let [old-state (get-in @state [:run :phase])
+        new-state (if (= :movement old-state)
+                    :encounter-ice
+                    old-state)]
+    (system-msg state side old-state)
```

Comment:
Are these supposed to be here?

Reply:
nope, but I bet it would have taken weeks for any player to notice :)